app/components/Countdown.tsx

```diff
@@ -0,0 +1,88 @@
+import * as moment from "moment";
+import * as React from "react";
+import * as zeroFill from "zero-fill";
+import * as styles from "./Countdown.scss";
+
+const numberFormatter = zeroFill(2);
+
+const SECOND = 1000;
+
+interface ICountdownComponentProps {
+  duration: moment.Duration;
+}
+
+// @todo remove moment prop checkers
```

Comment:
remove this comment please. I think it's not making now any sense.

Reply:
done